recipes/libiec61850/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+#include <format>
+#include <cstdlib>
+
+#include "libiec61850/iec61850_client.h"
+
+int main(int argc, char **argv)
+{
+    std::string hostname("localhost");
```

Comment:
Let's avoid hitting the network layer for the test_package. Calling a simple function should be enough